wrappingGsdk/wrapper/Program.cs

```diff
@@ -151,5 +189,54 @@ private static void LogMessage(string message)
             GameserverSDK.LogMessage(message);
         }
 
+        private static string ExecuteCommandAndCaptureOutput(string commandName, string arguments)
+        {
+            Process process = StartProcess(commandName, arguments);
+            string commandOut = process.StandardOutput.ReadToEnd();
+
+            try
+            {
+                process.WaitForExit(TimeSpan.FromSeconds(2).Milliseconds);
+            }
+            catch (Exception exp)
+            {
+                Console.WriteLine(exp.ToString());
+            }
+            return commandOut;
+        }
+
+        public static Process GetProcessPortInUse()
+        {
+            // execute netstat command for the given port
+            string commandArgument = string.Format("/c netstat -an -o -p tcp|findstr \":{0}.*LISTENING\"", _listeningPort);
```

Comment:
this command will fail on Linux. But we can leave it for a different PR, where we update this sample to work with Linux processes. In any case, please add a comment here that this works only for Windows

Reply:
Thanks - add the code that GetProcessPortInUse() is available only on Windows